app/views/incidents/show.pdf.prawn

```diff
@@ -164,14 +164,25 @@ prawn_document do |pdf|
     end
   end
 
-  pdf.move_cursor_to 160
+  if report.injured_passengers.present?
+    pdf.bounding_box [0, pdf.cursor], width: pdf.bounds.width, height: 20 do
+      pdf.move_down 5
+      pdf.text 'Injured Passenger Information',
+        align: :center, size: 12
+    end
+    report.injured_passengers.each.with_index 1 do |pax, index|
+      pdf.text "#{index}. #{pax.display_info}", size: 10
+    end
+  end
 
-  description = if report.long_description?
+  pdf.move_cursor_to 130
+
+  description = if report.long_description? || report.injured_passengers.count > 2
```

Comment:
Is this a change leftover from testing things out (`report.injured_passengers.count > 2`)?

Reply:
No - in either case we need to move the description to the next page.

Reply:
ok, looks good then